tapgarden/planter.go

```diff
@@ -485,7 +494,7 @@ func (c *ChainPlanter) gardener() {
 				continue
 			}
 
-			_, err := c.finalizeBatch()
+			_, err := c.finalizeBatch(nil)
```

Comment:
This path only gets hit in tests right?

Reply:
I didn't see exactly where the ticker was turned off currently.